stable/aerospike/Chart.yaml

```diff
@@ -1,14 +1,16 @@
-appVersion: v3.14.1.2
+appVersion: v4.5.0.5
 description: A Helm chart for Aerospike in Kubernetes
 name: aerospike
 keywords:
   - aerospike
   - big-data
 home: http://aerospike.com
-version: 0.2.1
+version: 0.2.3
 icon: https://s3-us-west-1.amazonaws.com/aerospike-fd/wp-content/uploads/2016/06/Aerospike_square_logo.png
 sources:
 - https://github.com/aerospike/aerospike-server
 maintainers:
 - name: kavehmz
   email: kavehmz@gmail.com
+- name: moeryomenko
```

Comment:
to add a new maintainer please first talk with the existing ones and one another pr for that

Reply:
remove